ava.config.js

```diff
@@ -1,3 +1,6 @@
+const semver = require("semver")
+
+const loader_arg = semver.lte(process.version, 'v20.5.0') ? '--loader=ts-node/esm' : '--import=@swc-node/register/esm-register'
```

Comment:
If we're going to migrate to swc (and generally I'm happy to) are there other uses of ts-node or other loaders that we can get rid of?

We already have too many compilers and builders. Most for agood reason, but this might be a good time and opportunity to consolidate a little

Reply:
Yeah, we can get rid of one. once, one(probably swc or tsx) works across the versions.

Reply:
Any insight as to which is better? Swc looks pretty cool, but then they all look pretty cool 🤔